controllers/user.controller.js

```diff
@@ -23,7 +24,21 @@ module.exports = {
         avatar,
       });
 
-      endpointResponse({ res, message: "Users was created", body: user });
+      const payload = {
+        password: user.password,
+        roleId: user.roleId
```

Comment:
You don't have to use user.roleId because in the line 16 you are using destructuring, Leave only roleId.